### PR TITLE
Docs: Add entry for Tempo query-type template variables

### PR DESCRIPTION
**Pull request by fabrizio-grafana**
Add entry for Tempo query-type template variables in "What's New In Grafana" for 10.2.

Trying the new process to update "What's New In Grafana", as discussed on Slack.

### CODE REVIEW

**Review by 09jvilla — APPROVED** [review]
Updated the description but now LGTM!

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -16,3 +16,14 @@ weight: -37
...
+## Support query-type template variables for Tempo data source
+
+<!-- Fabrizio Casati -->
+<!-- OSS, Enterprise -->
+
+_Generally available in Grafana Cloud_
+
+The Tempo data source now supports query-type template variables. With this, you can now create variables whose values are a list of attribute names or attribute values seen on spans received by Tempo.
+
+To learn more, refer to this [short video demo]( add link ) as well as to the [Variables page](https://grafana.com/docs/grafana/latest/dashboards/variables/) in the Grafana documentation.
```
Comment (09jvilla) [inline docs/sources/whatsnew/whats-new-next/index.md:29]: @fabrizio-grafana -- Once you add the link to your video demo, go ahead and merge!

**Review by 09jvilla — APPROVED** [review]
Approved pending the addition of the link to the demo video.

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -16,3 +16,16 @@ weight: -37
...
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
+
+## Support query-type template variables for Tempo data source
+
+<!-- Fabrizio Casati -->
+<!-- OSS, Enterprise -->
+
+_Generally available in Grafana Cloud_
+
+The Tempo data source now supports query-type template variables. With this, you can now create variables whose values are a list of attribute names or attribute values seen on spans received by Tempo.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:27]: We don't want to treat an inanimate thing as a person, so I've replaced "whose"
```suggestion
The Tempo data source now supports query-type template variables. With this update, you can create variables for which the values are a list of attribute names or attribute values seen on spans received by Tempo.
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -16,3 +16,16 @@ weight: -37
...
+
+To learn more, refer to the video demo below as well as to the [Variables page](https://grafana.com/docs/grafana/latest/dashboards/variables/) in the Grafana documentation.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:29]: Avoiding above/below for improved accessibility and tweaking the wording for docs link and using a partial URL
```suggestion
To learn more, refer to the following video demo, as well as the [Grafana Variables documentation](/docs/grafana/latest/dashboards/variables/).
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 4]
```diff
@@ -16,3 +16,16 @@ weight: -37
 # What’s new in Grafana Cloud
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
+
+## Support query-type template variables for Tempo data source
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:20]: Suggest shortening this a bit

```suggestion
## Query-type template variables for Tempo data source
```

**Review by imatwawana — COMMENTED** [review]
Thanks for being the first to update the Cloud What's new! I've made a few suggestions.

**Comment by 09jvilla** [comment]
Thanks for the review @imatwawana . Added in your changes.

**Review by imatwawana — APPROVED** [review]
LGTM!

**Pull request merged**